server/models/Request.js

```diff
@@ -0,0 +1,70 @@
+const mongoose = require("mongoose");
+
+const requestSchema = new mongoose.Schema(
```

Comment:
Please rebase/adjust the base of this PR to only reflect the changes pertaining to this ticket